readme.org

```diff
@@ -98,7 +98,7 @@ See [[#supported-keyboards][Supported Keyboards]] for details of supported keybo
 
 Specifies the ZMK board.
 
-For onboard controller keyboards (keyboards with an integrated controller), enter the keyboard name, e.g. ~ahokore~, ~ble_chiffre~, ~technikable~, ~zaphod~.
+For onboard controller keyboards (keyboards with an integrated controller), enter the keyboard name, e.g. ~ahokore~, ~ble_chiffre~, ~technikable~, ~zaphod~, ~willis~.
```

Comment:
This is just a small number of examples, not a complete list, so it doesn't need to be added here. Also, they are in alphabetical order.

Reply:
Removed the example, thank you. I was just following the changes from the `technikable` commit.

Reply:
Thanks! I thought I was keeping examples to 3, so I don't know why there was 4 there already!